update count label for clarity

parthsidpara opened the pull request:
Dashboard with updated label
<img width="1798" height="1085" alt="image" src="https://github.com/user-attachments/assets/dfec270a-e329-406a-bf3b-84f69acce4ec" />

parthsidpara committed:
update count label for clarity

parthsidpara committed:
Merge branch 'main' into ux/update-count-label

debloper commented:
Can you please update the "duration" label to "keep for" and "s" to "seconds"?

So that it reads like: "keep for [ ] seconds and switch every [ ] cycles"

Also, verify it works in mobile (smaller resolution portrait) layout.

parthsidpara committed:
update 'Duration' label to 'Keep for'

parthsidpara committed:
improve mobile responsiveness and visibility

parthsidpara committed:
update 's' to 'seconds'

parthsidpara commented:
I've updated the labels.
Also adjusted the CSS to improve visibility on smaller mobile layouts.
It works, but we can refine the mobile UI further after the v5 release.

parthsidpara committed:
fix inconsistent capitalization, standardize labels to title case

debloper reviewed (approved):
Seems good 👍
